2016-2017_Starstruck/Swerve Driver Proto/include/SwerveLib.h

```diff
@@ -19,7 +19,19 @@
 #endif
 
 // Default drive mode (MUST BE A GLOBAL DriveConfiguration LISTED BELOW)
-#define DEFAULT_DRIVE_MODE holonomicDrive
+#define DEFAULT_DRIVE_MODE tankDrive
+
+//LCD Constants
```

Comment:
There's prbbly a more dynamic way to do this but we'll worry about it after Rowan